content/en/docs/v3.6/learning/persistent-storage-files.md

```diff
@@ -0,0 +1,621 @@
+---
+title: etcd persistent storage files
+weight: 2650
+description: Reference of the persistent storage format and files
+---
+
+# etcd persistent storage files
```

Comment:
As above - in netlify preview this displays as a duplicate header so we can just remove it.